status.json

```diff
@@ -58,11 +58,16 @@
   {
     "name": "Same-site cookies",
     "category": "Security",
-    "link": "https://tools.ietf.org/html/draft-west-first-party-cookies-07",
+    "link": "https://tools.ietf.org/html/draft-ietf-httpbis-cookie-same-site-00",
     "summary": "Defines a SameSite attribute, allowing a server to specify that cookies should not be sent with cross-site requests.",
     "standardStatus": "Editor's draft",
     "ieStatus": {
-      "text": "In Development"
+      "text": "Shipped",
+      "ieUnprefixed": "11"
+    },
+    "ff_views": {
```

Comment:
This part can be removed as it come from [Chrome status](https://www.chromestatus.com/features/4672634709082112).

Reply:
What about IE status? We also shipped SameSite cookies in IE 11.